detox/src/Detox.js

```diff
@@ -127,6 +127,10 @@ class Detox {
     });
   }
 
+  deviceName() {
```

Comment:
Was there a specific urge to add a getter for the device name specifically to Detox object instance?

Usually we `export { detox, device, expect, ... }`, so what is the issue with simply accessing `device` and getting its name?

Reply:
👍

Reply:
Ahem, it seems that the code was left intact while you had resolved the conversation. Did you forget to push, maybe?